__tests__/helpers.ts

```diff
@@ -12,6 +12,7 @@ type CLICommands =
   | 'login'
   | 'logout'
   | 'preview-docs'
+  | 'verify-config'
```

Comment:
I'm not sure about the name though. Most likely, we don't want to name it `lint-config` as we already have such an option (so it will be confusing to use the same for a command name), but `verify-config` also doesn't sound right to me. What are other options? `config`, `validate-config`, `check-config`, ...?

Reply:
Yeah, `lint-config` was already in use, and that is the reason I have chosen the command name such as `verify-config`. I don't have a strict preference in this naming, so can be `validate-config` or smth else.
Let's discuss it to finalize.

cc: @lornajane , @RomanHotsiy , @adamaltman

Reply:
I think `check-config` could be more clear, and we can encourage users to run it as a CI "check" as well.

Reply:
@RomanHotsiy, @adamaltman any obections?

Reply:
'check-config' it is.
I will rename it soon. Thx